cms/envs/aws_migrate.py

```diff
@@ -1,30 +0,0 @@
-"""
```

Comment:
@edx/devops this change will require updates in tooling around edx-platform migrations.

@jibsheet will your changes conflict with this since you are working on the CSMH Extended which has two databases?